### PR TITLE
designed oops page

### PR DESCRIPTION
**Pull request by MalkySinowitz**

### CODE REVIEW

File: extension-ui/oops.scss
Comment (batiFinkelshtein) [inline extension-ui/oops.scss:1]: please change the px to%

**Review by batiFinkelshtein — APPROVED** [review]

**Review by batiFinkelshtein — APPROVED** [review]
the code show very good!

File: extension-ui/oops.html [hunk 1]
```diff
@@ -3,12 +3,18 @@
...
 </head>
 
 <body>
-  <div class="container">
-    <h1>Access to this site is blocked</h1>
-    <p>If you believe this is an error, please contact your administrator.</p>
+  <div>
+    <div class="content">
+      <h1>Oops...</h1>
+      <p>This site is blocked according to your current profile settings in timeOut</p>
```
Comment (e171271) [inline extension-ui/oops.html:13]: move all texts to constants file

File: extension-ui/oops.scss [hunk 2]
```diff
@@ -0,0 +1,122 @@
+body {
+  background-color: #f0f4f8;
```
Comment (e171271) [inline extension-ui/oops.scss:2]: the following comments relevant for all this file
1. colors and font-family should come from variables file
2. don't use px's
3. use scss structure
4. you can't write styles on elements like body, h1, p etc, you should add classes or id's and style them

**Review by e171271 — CHANGES_REQUESTED** [review]

**Review by batiFinkelshtein — APPROVED** [review]
you fix your wrongs!

**Pull request merged**